.github/workflows/test.yml

```diff
@@ -46,11 +46,11 @@ jobs:
     runs-on: ${{ matrix.os }}
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-java@v2
+      - uses: actions/setup-java@v3
```

Comment:
v3 is current (well, v3.3.0 is, but they move the v3 tag along with releases) https://github.com/actions/setup-java/tags